README.md

```diff
@@ -51,6 +52,19 @@ lexisnexis_instant_verify_workflow: "customers.gsa.instant.verify.workflow"
 lexisnexis_phone_finder_workflow: "customers.gsa.phonefinder.workflow"
 lexisnexis_timeout: "50"
 
+# Uncomment below once TrueID is supported by the fake server
+# doc_auth_vendor_randomize_percent: 95
+# doc_auth_vendor_randomize_alternate_vendor: 'lexisnexis'
```

Comment:
we don't need the randomize percent for this server, right?

Reply:
The "primary" is not TrueID currently in the IDP, so the alternate vendor stuff is still needed as I understand it

Reply:
right but the `doc_auth_vendor_randomize_percent` because the precent doesn't matter for this app? it just needs to support both cases?